SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -765,47 +776,49 @@ int32 USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConn
 
 				if (Channel)
 				{
-					// if it is relevant then mark the channel as relevant for a short amount of time
+					// SpatialGDK - Only replicate actors marked as relevant (rate limiting).
 					if (bIsRelevant)
 					{
+						// if it is relevant then mark the channel as relevant for a short amount of time
 						Channel->RelevantTime = Time + 0.5f * FMath::SRand();
-					}
-					// if the channel isn't saturated
-					if (Channel->IsNetReady(0))
-					{
-						// replicate the actor
-						UE_LOG(LogNetTraffic, Log, TEXT("- Replicate %s. %d"), *Actor->GetName(), PriorityActors[j]->Priority);
-						if (DebugRelevantActors)
-						{
-							LastRelevantActors.Add(Actor);
-						}
 
-						if (Channel->ReplicateActor())
+						// if the channel isn't saturated
+						if (Channel->IsNetReady(0))
 						{
-							ActorUpdatesThisConnectionSent++;
+							// replicate the actor
+							UE_LOG(LogNetTraffic, Log, TEXT("- Replicate %s. %d"), *Actor->GetName(), PriorityActors[j]->Priority);
 							if (DebugRelevantActors)
 							{
-								LastSentActors.Add(Actor);
+								LastRelevantActors.Add(Actor);
 							}
 
-							// Calculate min delta (max rate actor will update), and max delta (slowest rate actor will update)
-							const float MinOptimalDelta = 1.0f / Actor->NetUpdateFrequency;
-							const float MaxOptimalDelta = FMath::Max(1.0f / Actor->MinNetUpdateFrequency, MinOptimalDelta);
-							const float DeltaBetweenReplications = (World->TimeSeconds - PriorityActors[j]->ActorInfo->LastNetReplicateTime);
-
-							// Choose an optimal time, we choose 70% of the actual rate to allow frequency to go up if needed
-							PriorityActors[j]->ActorInfo->OptimalNetUpdateDelta = FMath::Clamp(DeltaBetweenReplications * 0.7f, MinOptimalDelta, MaxOptimalDelta);
-							PriorityActors[j]->ActorInfo->LastNetReplicateTime = World->TimeSeconds;
+							if (Channel->ReplicateActor())
+							{
+								ActorUpdatesThisConnectionSent++;
+								if (DebugRelevantActors)
+								{
+									LastSentActors.Add(Actor);
+								}
+
+								// Calculate min delta (max rate actor will update), and max delta (slowest rate actor will update)
+								const float MinOptimalDelta = 1.0f / Actor->NetUpdateFrequency;
```

Comment:
Is this protected against divide by zero?

Reply:
This is just a whitespace change and so should be fine.